File: leetcode/Question-6.js
// Question 5 -> Frequency Find Of Array 

let arr = [1, 2, 3, 2, 4, 1, 2, 5];

function findFrequencyArray(a) {
    let freq = {};

    for (let i = 0; i < a.length; i++) {

        if (freq[a[i]]) {
            freq[a[i]]++;
        } else {
            freq[a[i]] = 1;
        }
    }

    console.log(freq);
}

findFrequencyArray(arr);

File: leetcode/Question-6.cpp
// Question 5 -> Frequency Find Of Array
#include <iostream>
using namespace std;

int main()
{
    int arr[] = {1, 2, 3, 2, 4, 1, 2, 5};
    int n = 8;

    bool visited[100] = {false};

    for (int i = 0; i < n; i++)
    {
        if (visited[i])
            continue;

        int count = 1;

        for (int j = i + 1; j < n; j++)
        {
            if (arr[i] == arr[j])
            {
                count++;
                visited[j] = true;
            }
        }

        cout << arr[i] << " : " << count << endl;
    }

    return 0;
}
